rho/vault.py

```diff
@@ -52,7 +54,12 @@ def read_vault_password(vault_password_file):
 
 def get_vault_password(prompt=PROMPT):
     """Requests the users password from the command line """
-    return getpass("\033[01;36m" + prompt + "\033[0m")
+    vault_password = ''
+    while vault_password == '':
+        vault_password = getpass("\033[01;36m" + prompt + "\033[0m")
+        if vault_password == '':
```

Comment:
Doing this validation it would allow one or more spaces as a vault password, is it ok?

Reply:
I've verified that spaces are fine as a password.